test/browser-tests/router-get/router-get.js

```diff
@@ -0,0 +1,296 @@
+/*
+  Copyright 2016 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// This is a test and we want descriptions to be useful, if this
+// breaks the max-length, it's ok.
+
+/* eslint-disable max-len */
+/* eslint-env browser, mocha */
+/* global testHelper */
```

Comment:
Optional: If you think you'll be using `testHelper` throughout the tests under `/browser-tests/`, you could move this config into a separate directory-scoped `.eslintrc` file. (E.g. [this](https://github.com/GoogleChrome/samples/blob/gh-pages/service-worker/.eslintrc) from the Chrome samples project.)